debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/util/TestHelper.java

```diff
@@ -96,7 +96,7 @@
 
     public static JdbcConfiguration adminJdbcConfig() {
         return JdbcConfiguration.copy(Configuration.fromSystemProperties("database."))
-                .withDefault(JdbcConfiguration.DATABASE, "master")
+                .withDefault(SqlServerConnectorConfig.DATABASES_FIELD_NAME, "master")
```

Comment:
It's weird to have the JDBC configuration that expects a singular database name depend on higher-level configuration that provides a list of database names. It should be safe to not have the default here at all. The user will connect to their [default database](https://docs.microsoft.com/en-us/sql/t-sql/statements/alter-login-transact-sql?view=sql-server-ver15.